pkg/stream/message_test.go

```diff
@@ -19,9 +19,12 @@ package stream
 import (
 	"bytes"
 	"encoding/binary"
+	"io"
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/kubeedge/kubeedge/common/constants"
```

Comment:
```suggestion
	
	"github.com/kubeedge/kubeedge/common/constants"
```
Add a newline to fix lint error